docs/src/tutorials/run_circuit_virtual.md

```diff
@@ -0,0 +1,88 @@
+# Run a circuit on a Virtual QPU
+
+In the previous tutorial, we introduced some basic concepts of quantum computing, namely the quantum circuit and quantum gates. 
+
+We also learnt how to build a quantum circuit using `Snowflake` and simulate the result of such circuit using our local machine. 
+
+In this tutorial, we will the steps involved in running a quantum circuit on a both a virtual and also a real Quantum Processing Unit (QPU). 
+
+
+## QPU Object
+Interactions with different QPUs are facilitated using `struct`s (objects) that represent QPU hardware.  These structures are used to implement a harmonized interface, and are derived from an `abstract type` called `AbstractQPU`. This interface gives you a unified way to write code that is agnostic of the quantum service you are using. The interface dictates how to get metadata about the QPU, how to run a quantum circuit on the QPU, and more. 
+
+!!! warning 
+    You should not use `AbstractQPU`, rather use a QPU object which is derived from `AbstractQPU`. For further details on the implemented derived QPUs, see the [Library page](../library.md#Quantum-Processing-Unit). 
```

Comment:
Stop the sentence after `AbstractQPU` and before `rather` and start a new sentence with `Rather`.